hyperspy/io.py

```diff
@@ -105,13 +106,16 @@ def load(filenames=None,
         until it finds a name that is not yet in use.
     lazy : {None, bool}
         Open the data lazily - i.e. without actually reading the data from the
-        disk until required. Allows opening arbitrary-sized datasets. default
+        disk until required. Allows opening arbitrary-sized datasets. The default
         is `False`.
+    convert_units : {None, bool}
```

Comment:
I am not sure if it is a good idea to let the reader decide if to convert units or not. I would rather default to ``False`` and, if the user finds that the units are not convenient, he can always call the ``AxesManager.convert_units`` manually.

Reply:
This is actually one of the motivation of this PR: for example, the fei reader convert scale and offset systematically to "nm" for convenience, because the file is saved with the units in "m".  Same for the bruker reader, the units are saved in "um" in the file, which is not great for nanometer scale EDS mapping. 
Allowing to have a reader parameter to determine if the units needs to be converted or not is an improvement to the current situation. Only the reader, which present this issue have `auto_convert_units=True`.

Reply:
I still think that it is better to default to "don't autoconvert units" and not to hardwire any different behaviour as this is not transparent for the user. Units auto conversion is convenient, but it is somehow unpredictable: it can change with different hyperspy and pint versions and and files stored in the same format could return different units depending on the data stored—too much unpredictability IMO.

Personally I think that the units in the file must be respected i.e. if FEI writes everything in S.I. we should read the data in S.I. by default. I know that this is not the current situations, and for me this fixes it as long as it respects the file units by default. The reason why some file readers automatically convert units is that there was no easy way of converting units before you implemented this feature.

Reply:
Since it is only the FEI reader and some tiff files which can generate inappropriate units, I removed the auto_convert_units reader parameter and I reverted to the previous conversion in the FEI reader, since I would be annoyed to have to write `convert_units=True` each time I want to load an emi/ser file.